docs/designers-developers/developers/block-api/block-edit-save.md

```diff
@@ -390,15 +390,15 @@ When the editor loads, all blocks within post content are validated to determine
 
 If a block is detected to be invalid, the user will be prompted to choose how to handle the invalidation:
 
-![Invalid block prompt](https://user-images.githubusercontent.com/8876600/87853743-bb45db80-c947-11ea-8e61-93d948d2ba84.png)
+![Invalid block prompt](https://user-images.githubusercontent.com/7753001/88754471-4cf7e900-d191-11ea-9123-3cee20719d10.png)
 
-Clicking Resolve button will open Resolve Block dialog box with two buttons:
+Clicking **Attempt Block Recovery** button will attempt recovery action as much as possible.
 
-- **Convert to HTML**: Protects the original markup from the saved post content and convert the block from its original type to the HTML block type, enabling the user to modify the HTML markup directly.
-- **Convert to Blocks**: Protects the original markup from the saved post content and convert the block from its original type to the validated block type.
-
-Clicking the "3-dot" menu on the side of the block displays two options:
+Clicking the "3-dot" menu on the side of the block displays three options:
 
+- **Resolve**: Open Resolve Block dialog box with two buttons:
+  - **Convert to HTML**: Protects the original markup from the saved post content and convert the block from its original type to the HTML block type, enabling the user to modify the HTML markup directly.
+  - **Convert to Blocks**: Protects the original markup from the saved post content and convert the block from its original type to the validated block type.
 - **Convert to Classic Block**: Protects the original markup from the saved post content as correct. Since the block will be converted from its original type to the Classic block type, it will no longer be possible to edit the content using controls available for the original block type.
 - **Attempt Block Recovery**: Attempts recovery action as much as possible.
```

Comment:
Well done spotting this bit of documentation, something I'd have probably missed!

It looks like this line will need to be removed as it duplicates the one above..

Reply:
Ahh! Missed it! Thx!